core/src/main/java/org/jboss/pnc/causeway/pncclient/PncBuild.java

```diff
@@ -2,9 +2,34 @@
 
 import org.jboss.pnc.causeway.rest.BrewNVR;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public final class PncBuild {
 
+    public final List<PncArtifact> buildArtifacts;
+    public final List<PncArtifact> dependencies;
+
+    public PncBuild() {
+        buildArtifacts = new ArrayList<>();
+        dependencies = new ArrayList<>();
+    }
+
     public BrewNVR createNVR() {
         return null;
     }
+
+    public static class PncArtifact {
```

Comment:
The reason is that I have class structure under control and I can have only attributes / values I need. For example type I would expect to be maven / npm / etc as opposed to type used in PNC (build result / build dependency). Other reason is that it isolates me from PNC rest model class changes